docs/modules/ROOT/pages/tutorials/deploy_kind.adoc

```diff
@@ -78,6 +78,23 @@ This error happens when ArgoCD bootstraps itself.
 
 This happens because the Argo CD server pod is redeployed and the Argo CD Terraform provider loses connection. You can simply re-run the command `terraform apply` to finalize the bootstrap of the cluster.
 
+==== loki-stack-promtail pods stuck with status CrashLoopBackOff
+
+When the pods of loki-stack-promtail are stuck in a creation loop with the following logs:
+
+[source,]
+----
+level=error ts=2023-05-09T06:32:38.495673778Z caller=main.go:117 msg="error creating promtail" error="failed to make file target manager: too many open files"
+Stream closed EOF for loki-stack/loki-stack-promtail-bxcmw (promtail)
+----
+
+You will have to increase the upper limit on the number of INotify instances that can be created per real user ID:
+
+[source,bash]
+----
+sudo echo '512' > /proc/sys/fs/inotify/max_user_instances
+----
```

Comment:
I found out that this command only applies temporarily and is reset on every session, at least that was my case. Alternatively, I think we can properly change the value with the command `sudo sysctl fs.inotify.max_user_instances=512`.

Reply:
Sorry, my bad! Actually this command also changes the value temporarily. If we want to have a permanent change, you'll need to edit the `/etc/sysctl.conf` (see [here](https://www.suse.com/support/kb/doc/?id=000020048) for reference).

Reply:
We could propose both, if some users want to test the DevOps stack without changing their computer config ?

Reply:
That was what I was going to propose 😉